packages/contracts/deploy/10_offchain_resolver.js

```diff
@@ -1,18 +1,20 @@
 const { ethers } = require("hardhat");
 
-module.exports = async ({deployments}) => {
+module.exports = async ({getNamedAccounts, deployments, network}) => {
     const {deploy} = deployments;
-    const signers = await ethers.getSigners();
-    const owner = signers[0].address;
-    const resolver = await deploy('OffchainResolver', {
-        from: owner,
-        args: ['http://localhost:8000/{sender}/{data}.json', ['0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266']],
+    const {deployer, owner} = await getNamedAccounts();
+    if(!network.tags.demo){
```

Comment:
You can avoid the need for this by setting the appropriate account indexes for test networks in hardhat.config.js.

Reply:
I just flipped the order of deployer and owner so that owner is always `accounts[0]` which shouldn't break any other dependencies.